db/malloc_stats.cc

```diff
@@ -13,18 +13,15 @@
 #include <memory>
 #include <string.h>
 
-namespace rocksdb {
+#include "port/jemalloc_helper.h"
 
-#ifdef ROCKSDB_JEMALLOC
-#ifdef __FreeBSD__
-#include <malloc_np.h>
-#else
-#include "jemalloc/jemalloc.h"
 #ifdef JEMALLOC_NO_RENAME
```

Comment:
This ifdef is now out of a conditional like `ifdef ROCKSDB_JEMALLOC && !(__FreeBSD__)`. I hope this is ok.